frontend/javascripts/oxalis/constants.js

```diff
@@ -141,6 +143,8 @@ const Constants = {
   BUCKET_WIDTH: 32,
   BUCKET_SIZE: 32 ** 3,
   VIEWPORT_WIDTH,
+  // About the area the brush reaches at maximum radius (pi * 300 ^ 2).
+  AUTO_FILL_AREA_LIMIT: 200000,
```

Comment:
Good point. Also, I'd think that we can go higher than a brush radius, since the brush has to be drawn on every mouse movement, whereas the trace tool works in "atomic steps" (where it's okay to wait a bit). So, maybe we can go as far as "10 brush strokes".

Reply:
Agreed :+1: